src/components/EdgeLink.svelte

```diff
@@ -24,9 +26,9 @@
 <ObsidianLink
 	{plugin}
 	{display}
-	path={edge.target_id}
-	resolved={edge.target_attr.resolved}
-	cls="{cls} BC-edge {edge.attr.explicit
+	path={target_node.path}
+	resolved={target_node.resolved}
+	cls="{cls} BC-edge {edge.implied
 		? 'BC-edge-explicit'
-		: `BC-edge-implied BC-edge-implied-${edge.attr.implied_kind}`}"
+		: `BC-edge-implied BC-edge-implied-${ 'transitive' /*edge.attr.implied_kind */}`}"
```

Comment:
`implied_kind` isn't _just_ `"transitive"`. It's `"transitive:${stringify_transitive_rule(rule)}"`